device/accton/x86_64-accton_as4630_54te-r0/plugins/psuutil.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
These platform plugins have been deprecated in favor of the new sonic-platform package. Please refer to the documentation here: https://github.com/Azure/SONiC/wiki/Porting-Guide#sonic-platform-api-package